README.md

```diff
@@ -0,0 +1,195 @@
+## Creating the DUH document
+NOTE: this tutorial was made using version 1.15.0 of DUH
+
+The DUH document is a JSON5 file that describes an IP block similar to IPXACT.
```

Comment:
This is significant understatement of what DUH document is. There are several types of DUH documents. DUH component is just one of them.